### DISCUSSION TITLE
Localization - How?

### DISCUSSION THREAD

**Discussion by synsyst** [GuavaCZ/filament-knowledge-base#65] [Q&A] [answered]
How do I modify localization in this package? I see some lang files, but they're hardcoded to languages

1. How do I even enable them? They take locale from app.config?
2. Can we get a en file with helper functions instead so this works with our own translation json files? As it is now it seems I need to extend this package or edit directly in the vendor files to make it work. I'm fairly new at Filament, so if I'm missing something I'd love some input

**Comment by lukas-frey** [comment, marked as answer]
If the package doesn't support the language you want, then you can publish the language files to your project and add your own language/translations.

https://laravel.com/docs/11.x/localization#overriding-package-language-files
